PowerSession.Main/Commands/RecordSession.cs

```diff
@@ -47,7 +47,7 @@ internal IEnumerable<SessionLine> Lines()
                 if (lineData.Count != 3) throw new InvalidDataException("Invalid record data");
                 var rv = new SessionLine
                 {
-                    Timestamp = (double) lineData[0],
+                    Timestamp = Convert.ToDouble(lineData[0]),
```

Comment:
Any valid numerical will now be converted to a double. For example when processing with PowerShell the `ConvertTo-Json` function will remove any non-significant trailing zeros.

Reply:
could you be more specific at this, or put an example corner case here https://rextester.com/TFFW46728 ?

Reply:
@ibigbug if I post-process the cast file and my JSON serializer emits 1 rather 1.23456 as a the Timestamp (because I've rebased time) on an event then PowerSession will fail with an exception on the cast of an int64 -> double.

`Convert.ToDouble` will convert any integer, float, double or any .NET numerical type to double without the need for a cast.

Reply:
that's the same as what I assumed. but the link I post above does cast `1` to `double` with no issue.

And also I don't see issue casting int64 to float which has better precision.

what's the issue you are facing exactly?

can you please share an example like above?

Reply:
test.cast:
```json
{"version":2,"width":160,"height":25,"timestamp":1618170765,"env":{"POWERSESSION_RECORDING":"1","SHELL":"powershell.exe","TERM":"NotSure"},"Valid":true}
[0,"o","\u001b[2J\u001b[m\u001b[HWindows PowerShell\r\nCopyright (C) Microsoft Corporation. All rights reserved.\r\n\r\nTry the new cross-platform PowerShell https://aka.ms/pscore6\u001b]0;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe\u0007\u001b[?25h"]
```

run with:
```pwsh
PS C:\Users\Craig\> PowerSession-win10-x64.exe play .\test.cast
Unhandled exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.InvalidCastException: Unable to cast object of type 'System.Int64' to type 'System.Double'.
   at PowerSession.Commands.RecordSession.Lines()+MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at PowerSession.Commands.RecordSession.StdoutRelativeTime()+MoveNext()
   at PowerSession.Commands.PlayCommand.Execute()
   at PowerSession.Cli.Program.<>c.<Main>b__0_1(FileInfo file)
```

Reply:
LGTM